### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -2092,6 +2098,16 @@ msgid ""
 "different remotes' fetch refspecs map into different\n"
 "tracking namespaces."
 msgstr ""
+"Існує декілька віддалених призначень, специфікатори отримування посилань "
```
Comment (kasya) [inline po/uk.po:2101]: 👏🏼

**Review by kasya — COMMENTED** [review]
LGTM!

File: po/uk.po [hunk 2]
```diff
@@ -2047,21 +2047,27 @@ msgid ""
 "After fixing the error cause you may try to fix up\n"
 "the remote tracking information by invoking:"
 msgstr ""
+"\n"
+"Після усунення причини помилки ви можете спробувати виправити\n"
+"інформацію про віддалене відстежування викликавши:"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"інформацію про віддалене відстежування здійснивши виклик:"
```

Comment (arkid15r) [reply po/uk.po]: nice one!

**Pull request merged**